README.md

```diff
@@ -1,6 +1,22 @@
 Zonemaster LDNS
 ===============
 
+**Table of contents**
+
+* [Introduction](#introduction)
+  * [Dependencies](#dependencies)
+  * [Compatibility](#compatibility)
```

Comment:
The indentation of the Dependencies and Compatibility items don't match the heading levels of the actual headings. I think it makes more sense to have them on the same level as introduction and installation, rather than having them as part of the introduction. Would you consider outdenting the Dependencies and Compatibility items in the TOC?

Reply:
My mistake. Corrected TOC. I also merged "dependencies" and "compatibility" as proposed by @JohanIIS  since the latter is so small.